ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -256,7 +256,7 @@ public long getPropagationManagerGetBlockTimeoutMillis() {
 
   public static class Builder {
     private SyncMode syncMode = SyncMode.FULL;
-    private int fastSyncMinimumPeerCount = DEFAULT_FAST_SYNC_MINIMUM_PEERS;
+    private int syncMinimumPeerCount = DEFAULT_SYNC_MINIMUM_PEERS;
```

Comment:
If we change the name, shouldn't we also create a new flag without "fast" and the old one will be depreciated with the idea of ​​removing it later

Reply:
Yeah the external name was actually changed a few months ago. This is just me tidying up the name of the internal variable to match the new `--sync-min-peers` flag